vendor/bat-native-ads/src/bat/ads/internal/eligible_ads/ad_notifications/eligible_ad_notifications_v1.cc

```diff
@@ -158,54 +158,31 @@ void EligibleAdsV1::GetForUntargeted(
 CreativeAdNotificationList EligibleAdsV1::FilterCreativeAds(
     const CreativeAdNotificationList& creative_ads,
     const AdEventList& ad_events,
-    const BrowsingHistoryList& browsing_history) const {
+    const BrowsingHistoryList& browsing_history) {
   if (creative_ads.empty()) {
     return {};
   }
 
   CreativeAdNotificationList eligible_creative_ads = creative_ads;
 
+  frequency_capping::ExclusionRules exclusion_rules(
+      subdivision_targeting_, anti_targeting_resource_, ad_events,
+      browsing_history);
+  eligible_creative_ads = ApplyFrequencyCapping(
+      eligible_creative_ads, last_served_ad_, &exclusion_rules);
+
```

Comment:
Frequency capping must occur before filtering creative ads for seen ads/advertisers for round robin to work as expected